exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
+library(tidyverse)
+library(ggthemes)
+
+#Load malaria data Rstudio
+
+malaria_data <- read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/malaria.dat",header=TRUE)
+
+#View malaria data
+
+malaria_data
+
+#Assign time and cases as per the data
+
+time <- malaria_data$Time
+
+cases <- malaria_data$Cases
+
+rain <- malaria_data$Rain
```

Comment:
These are all correct syntax but I am wondering why you need these lines of code? What are they meant to do? And how do they answer the task/question at hand? When I go down your next lines of code, you don't even use these objects? So, why create them? What was your plan?